pkg/globalmanager/incrementallearningjob.go

```diff
@@ -305,6 +306,54 @@ func (jc *IncrementalJobController) sync(key string) (bool, error) {
 	return forget, err
 }
 
+// setWorkerNodeName sets worker nodeName for downstream
+func (jc *IncrementalJobController) setWorkerNodeName(job *sednav1.IncrementalLearningJob, jobStage string, nodeName string) error {
+	key := AnnotationsKeyPrefix + jobStage
+
+	ann := job.GetAnnotations()
+	if ann != nil {
+		if v, ok := ann[key]; ok && v == nodeName {
+			return nil
+		}
+	}
+
+	var err error
+	path := "/metadata/annotations/" + key
+	data := []struct {
+		Op    string `json:"op"`
+		Path  string `json:"path"`
+		Value string `json:"value"`
+	}{{"add", path, nodeName}}
+
+	var dataStr []byte
+	dataStr, err = json.Marshal(data)
```

Comment:
```suggestion
	dataStr, err := json.Marshal(data)
```

Reply:
don't need it

Reply:
var err error

Reply:
the above two definitions can be removed